### DISCUSSION TITLE
Using nftables' cluster match Instead of CLUSTERIP in strongSwan HA Configuration

### DISCUSSION THREAD

**Discussion by YEONGORI** [strongswan/strongswan#2364] [Q&A]
Hello,

I am currently setting up a high availability (HA) configuration for software tunnels based on strongSwan 5.9. The base OS is a custom OS based on RHEL8, and I am facing difficulties due to the absence of the CLUSTERIP module. Instead, I am looking for a way to use the cluster match module of nftables.

I am wondering if there is a way to configure strongSwan's HA setup using nftables' cluster match module.

**Comment by tobiasbrunner** [comment]
The [kernel patch](https://docs.strongswan.org/docs/5.9/features/highAvailability.html#_kernel_implementation) relies on CLUSTERIP. For newer kernels (6.3+), it now actually includes the CLUSTERIP module as it was removed from the kernel sources.
